### DISCUSSION TITLE
Levi-Civita symbol product identity

### DISCUSSION THREAD

**Discussion by ElJefeDelDesierto** [FeynCalc/feyncalc#291] [Q&A] [answered]
Hello great FeynCalc team.

There exists an identity relating the product of two Levi-Civita symbols (with no contracted indices) to the determinant of a matrix made of metric tensors. The identity is the following
![epsilonepsilonidentity](https://github.com/user-attachments/assets/12dba086-0cc7-4ac6-8a98-259cf559d737)
The same identity is shown in [Wikipedia](https://en.wikipedia.org/wiki/Levi-Civita_symbol). I have proven that this identity holds component by component (by using a python code). My question is: Is it possible to ask FeynCalc to use this identity to replace the product of Levi-Civita symbols in an expression? If so, how?

Thanks in advance!

**Comment by vsht** [comment, marked as answer]
Hi,

Contract does that automatically

https://feyncalc.github.io/FeynCalcBookDev/Contract.html

Am 20.12.24 um 03:51 schrieb ElJefeDelDesierto:

**Reply by ElJefeDelDesierto** [reply]
Thanks dear @vsht !!
